### PR TITLE
doc: add phrases on recommending tags of addon helpers

### PR DESCRIPTION
**Pull request by legendecas**
Suggesting tag addon helpers with `node-addon-api` on npm.

Fixes https://github.com/nodejs/node-addon-api/issues/670.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,16 +170,26 @@ npm run-script benchmark
...
-See [CONTRIBUTING.md](CONTRIBUTING.md) for more details on our philosophy around extending this module.
+<a name="resources"></a>
 
 ### **More resource and info about native Addons**
 - **[C++ Addons](https://nodejs.org/dist/latest/docs/api/addons.html)**
 - **[N-API](https://nodejs.org/dist/latest/docs/api/n-api.html)**
 - **[N-API - Next Generation Node API for Native Modules](https://youtu.be/-Oniup60Afs)**
 
+As node-addon-api's core mission is to expose the plain C N-API as C++
+wrappers, tools that facilitates n-api/node-addon-api to provide more
```
Comment (gabrielschulhof) [inline README.md]:
```suggestion
wrappers, tools that facilitate n-api/node-addon-api providing more
```

File: README.md [hunk 2]
```diff
@@ -172,16 +170,26 @@ npm run-script benchmark
...
+convenient patterns on developing a Node.js add-on with n-api/node-addon-api
```
Comment (gabrielschulhof) [inline README.md]:
```suggestion
convenient patterns on developing a Node.js add-ons with n-api/node-addon-api
```

File: README.md [hunk 3]
```diff
@@ -172,16 +170,26 @@ npm run-script benchmark
...
+can be published to NPM as standalone packages. It is also recommended to tag
+the package with `node-addon-api` to expose more visibilities on community.
```
Comment (gabrielschulhof) [inline README.md]:
```suggestion
such packages with `node-addon-api` to provide more visibility to the community.
```

File: README.md [hunk 4]
```diff
@@ -172,16 +170,26 @@ npm run-script benchmark
...
+
+Quick links to NPM searches: [keywords:node-addon-api](https://www.npmjs.com/search?q=keywords%3Anode-addon-api).
+
+## **Contributing**
+
+We love contributions from the community to **node-addon-api**.
```
Comment (gabrielschulhof) [inline README.md]:
```suggestion
We love contributions from the community to **node-addon-api**!
```

**Comment by mhdawson** [comment]
Landed as 89e62a9154b6057ed086a227db25caf26ac8c92a

**Pull request closed without merging**